tilings/strategies/verification.py

```diff
@@ -196,6 +196,11 @@ def get_genf(
     ) -> Expr:
         if not self.verified(tiling):
             raise StrategyDoesNotApply("tiling not locally verified")
+        if len(tiling.obstructions) == 1 and tiling.obstructions[0] in (
+            GriddedPerm.single_cell(Perm((0, 1, 2)), (0, 0)),
+            GriddedPerm.single_cell(Perm((2, 1, 0)), (0, 0)),
+        ):
+            return LocalEnumeration(tiling).get_genf(funcs=funcs)
```

Comment:
This (and the other bits of code) are adequately handling the positive case as well?

Reply:
Yeah, the 'magic' happens on lines 99 to 113.